### DISCUSSION TITLE
数据库要比较长时间才能恢复，应该怎么处理？

### DISCUSSION THREAD

**Discussion by gwhzh21** [dotnetcore/FreeSql#1423] [Q&A]
假设数据库要1-2个小时才能恢复。但是客户在5分钟内，就有3K多次访问。这样会导致线程池就挂了。在数据库恢复之后。重连很费时间，我在想，有没有什么熔断机制?应该在那块做熔断。

**Comment by 2881099** [comment]
FreeSqlBuilder  .UseConnectionString 默认就是这个机制
